llvm/test/CodeGen/RISCV/rvv/stack-folding.ll

```diff
@@ -1,6 +1,6 @@
 ; NOTE: Assertions have been autogenerated by utils/update_llc_test_checks.py UTC_ARGS: --version 5
-; RUN: llc < %s -mtriple=riscv32 -mattr=+v -verify-machineinstrs | FileCheck --check-prefixes=CHECK,RV32 %s
-; RUN: llc < %s -mtriple=riscv64 -mattr=+v -verify-machineinstrs | FileCheck --check-prefixes=CHECK,RV64 %s
+; RUN: llc < %s -mtriple=riscv32 -mattr=+v,+zfhmin,+zfbfmin,+zvfh,+zvfbfmin -verify-machineinstrs | FileCheck --check-prefixes=CHECK,RV32 %s
```

Comment:
Can you add (back) a plain vector line so that we have coverage of trying to fold the case without the instructions?  (I think this just goes the integer path, but it be good to have coverage.)

Reply:
We need +zvfhmin and +zvfbfmin or we'll crash when trying to handle the scalable half/bfloat vectors. But we can still test the integer path if we make the scalar types illegal so I've added more run lines without zfhmin and zfbfmin